components/ui/search/LogoAnimated.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <svg
```

Comment:
As this is in a separate component, and the remain element, there is no need to inline it.

Reply:
I'm not sure what you mean. I could use Nuxt to separate each of the SVGs into the image folder. I avoided that because, in reality, the individual SVGs (that make up the outer & inner rings + the wrench) will only be seen together, on top of each other.

Reply:
I just wanted to say that this was in a good spot, no need to move it.

Reply:
Oh, okay